android/src/main/res/values-pt/strings.xml

```diff
@@ -0,0 +1,192 @@
+    <string name="label_time_span_minutes_ago">%1s minutos atrás</string>
+    <string name="label_time_span_hours_ago">%1s horas atrás</string>
+    <string name="label_time_span_days_ago">%1s dias atrás</string>
+    <string name="error_camera">Erro ao inicializar a câmera. Por favor verifique suas configurações de privacidade.</string>
+
+    <!-- Wallet -->
+    <string name="label_tokens">tokens</string>
+    <string name="label_more_tokens">mais tokens</string>
+    <string name="label_unconfirmed">não confirmado</string>
+    <string name="button_receive">Receber</string>
+    <string name="button_send">Enviar</string>
+    <string name="button_display_currency">Moeda: %1s</string>
+    <string name="label_last_sync">sincronizado %1s</string>
+    <string name="label_erg_price">1 ERG ≈</string>
+    <string name="label_fiat_amount">≈ %1s</string>
+    <string name="label_wallet_default">Minha carteira</string>
+
+    <!-- add wallet -->
+    <string name="menu_add_wallet">Adicionar carteira</string>
+    <string name="label_create_wallet">Nova carteira</string>
+    <string name="desc_create_wallet">Cria uma carteira (chaves e um endereço) a partir de uma frase mnemônica gerada aleatoriamente.</string>
+    <string name="label_restore_wallet">Restaurar carteira</string>
+    <string name="desc_restore_wallet">Restaura as chaves e endereço de uma carteira a partir de uma frase mnemônica.</string>
+    <string name="label_import_wallet">Importar carteira</string>
+    <string name="desc_import_wallet">Use um arquivo de carteira exportado ou salvo para importar uma carteira existente (chaves e endereço).</string>
+    <string name="label_readonly_wallet">Carteira em modo Somente-Leitura</string>
+    <string name="desc_readonly_wallet">Adiciona um endereço de carteira em modo somente-leitura. Não salvará as chaves no aparelho.</string>
+
+    <!-- add readonly wallet -->
+    <string name="intro_add_readonly">Entre seu endereço público de carteira para adicionar a carteira em modo somente-leitura.</string>
+    <string name="label_wallet_address">Endereço de carteira</string>
+
+    <!-- Create wallet -->
+    <string name="intro_create_wallet">Está é a sua frase mnemônica de 15 palavras. Estas palavras também são conhecidas como "palavras-semente" ou 
+        "chave de papel".
+        Com essas 15 palavras, você pode restaurar sua carteira em outro aparelho se perder acesso a este.\nÉ crucial
+    que você escreva cuidadosamente todas as 15 palavras (com caneta e papel) e guarde-as em um local seguro.</string>
+    <string name="intro_confirm_create_wallet">Para confirmar que sua frase escrita a mão estão correta, digite agora duas palavras da frase:</string>
+    <string name="check_confirm_create_wallet">Eu entendo que sou responsável por armazenar minhas frases mnemônicas em um local seguro.
+    É a unica maneira de acessar minha carteira se as chaves armazenadas neste aparelho forem perdidas ou se tornarem inacessíveis.</string>
```

Comment:
My IDE told me this should be "única" - don't know how good the portuguese spell checking is, could you make a remark on this?